src/char/char_logif.cpp

```diff
@@ -566,7 +568,7 @@ int chlogif_parse_askkick(int fd, struct char_session_data* sd){
 			}
 			else
 			{// Manual kick from char server.
-				struct char_session_data *tsd;
+				struct char_session_data *tsd = NULL;
```

Comment:
This doesn't have to be initialized to null because we do direct assignments in the ARR_FIND with no pre or post checks.